app/ad-dice.js

```diff
@@ -3,22 +3,26 @@
 const angular = require("angular");
 const _ = require("lodash");
 
-angular.module("acey-deucey").directive("dice", function() {
+angular.module("acey-deucey").directive("adDice", function() {
     return {
-        template: `<div class="align-center v-align grid-block" ng-click="rollDice()">
-                        <div class="shrink align-center grid-block">
+        template: `<div class="align-center v-align grid-block">
+                        <div class="align-center grid-block" ng-if="!rolls.first">
+                            <button class="button" ng-click="rollDice()">Roll dice!</button>
+                        </div>
+                        <div class="shrink align-center grid-block" ng-if="rolls.first">
```

Comment:
Possible cleanup: as we discussed earlier, instead of having `ng-if` on both dice `div`s, create a single parent wrapper `div` and have one `ng-if` there. This will of course change the styling around a bit.